zebra-chain/src/parameters/network/subsidy.rs

```diff
@@ -289,6 +289,26 @@ const POST_NU6_FUNDING_STREAM_START_HEIGHT_MAINNET: u32 = 2_726_400;
 /// The start height of post-NU6 funding streams on Testnet as described in [ZIP-1015](https://zips.z.cash/zip-1015).
 const POST_NU6_FUNDING_STREAM_START_HEIGHT_TESTNET: u32 = 2_976_000;
 
+/// The one-time lockbox disbursement output addresses and amounts expected in the NU6.1 activation block's
+/// coinbase transaction on Mainnet.
+// TODO: Update this with hard-coded values once the ZIP is finalized and uncomment
+//       `sum_of_one_time_lockbox_disbursements_is_correct`.
+pub const NU6_1_LOCKBOX_DISBURSEMENT_MAINNET: [(&str, i64); 0] = [];
+
+/// The one-time lockbox disbursement output addresses and amounts expected in the NU6.1 activation block's
+/// coinbase transaction on Testnet.
+// TODO: Update this with hard-coded values once the ZIP is finalized and uncomment
+//       `sum_of_one_time_lockbox_disbursements_is_correct`.
+pub const NU6_1_LOCKBOX_DISBURSEMENT_TESTNET: [(&str, i64); 0] = [];
+
+/// The expected total amount of the one-time lockbox disbursement on Mainnet.
+// TODO: Add a reference to the ZIP and update this value if needed.
+pub const EXPECTED_NU6_1_LOCKBOX_DISBURSEMENT_TOTAL_MAINNET: i64 = 69_300;
```

Comment:
Are these denominated in zatoshis or ZEC? Also this PR should not be merged until these values are consistent with `NU6_1_LOCKBOX_DISBURSEMENT_MAINNET`

Finally, why is this a signed integer? Ideally, add an "unsafe" const constructor for `Amount` that panics if the amount is out of range, e.g. https://github.com/zcash/librustzcash/blob/a1c9aa12892a7c7e86365aa89990a584a634f53c/components/zcash_protocol/src/value.rs#L48-L51

Reply:
Suggestion from @str4d: Add a unit test that recomputes the lockbox value using the consensus machinery, and verify that it equals this constant.